.travis.yml

```diff
@@ -1,22 +1,34 @@
+# The testing matrix considers ruby/puppet versions supported by SIMP and PE:
+# ------------------------------------------------------------------------------
+#  release    pup   ruby      eol
+# PE 2016.4   4.7   2.1.9  TBD (LTS)
+# PE 2016.5   4.8   2.1.9  2017-10-31
+# SIMP6.0.0   4.8   2.1.9  TBD
+# PE 2017.1   4.9   2.1.9  2017-10-31
+# PE 2017.2   4.10  2.1.9  TBD
 ---
 language: ruby
 sudo: false
 cache: bundler
 before_script:
 - bundle update
+bundler_args: "--without development system_tests"
 before_install: rm Gemfile.lock || true
 script:
-  - bundle exec rake spec
+- bundle exec rake metadata_lint
+- bundle exec rake lint
+- bundle exec rake spec
 notifications:
   email: false
 rvm:
-  - 1.8.7 # CentOS 6
```

Comment:
I don't think the travis.yml should be changed

Reply:
because...

Reply:
The code in this repo runs in the native ruby, not the puppetlabs vendored ruby, so we need to test it against everything we can. Also, this is not a puppet module so testing against different versions of puppet won't get us anything